### PR TITLE
Splunk fetch bug

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21430

## Screenshots
The bug:
image

after fix: 
image

## Does it break backward compatibility?
   - [ ] No

### CODE REVIEW

File: dev-requirements-py3.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 flake8>=3.7.8
-demisto-sdk==0.3.4
+git+https://github.com/demisto/demisto-sdk.git@restructure
```
Comment (ShahafBenYakir) [inline dev-requirements-py3.txt]: remove this (need to pull from master to get latest version)

File: .circleci/config.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - restore_cache:
-          key: venv-{{ checksum "dev-requirements-py2.txt" }}-{{ checksum "dev-requirements-py3.txt" }}-{{ checksum ".circleci/build-requirements.txt" }}
+          key: venv-{{ checksum "dev-requirements-py2.txt" }}-{{ checksum ".circleci/build-requirements.txt" }}
```
Comment (ShahafBenYakir) [inline .circleci/config.yml]: What is that related?

File: Integrations/SplunkPy/SplunkPy.py [hunk 3]
```diff
@@ -187,7 +187,7 @@ def notable_to_incident(event):
     if demisto.get(event, "_time"):
         incident["occurred"] = event["_time"]
     else:
-        incident["occurred"] = datetime.now()
+        incident["occurred"] = datetime.now().strftime('%Y-%m-%dT%H:%M:%S.0+00:00')
```
Comment (ShahafBenYakir) [inline Integrations/SplunkPy/SplunkPy.py]: What is the `0+00:00` after the `%S`?

File: .circleci/config.yml (hunk 2 above)
Comment (Shellyber) [reply .circleci/config.yml]: Some changes made by Gal Rabin to test his fix. Reverting this

File: Integrations/SplunkPy/SplunkPy.py (hunk 3 above)
Comment (Shellyber) [reply Integrations/SplunkPy/SplunkPy.py]: Seems to be needed after checking and testing the bug. so thay Splunk will be able to parse the timestamp

**Comment by kirbles19** [comment]
@Shellyber Done.
